sample/src/main/java/com/polidea/cockpit/sample/VariantIndependentBaseMainActivity.kt

```diff
@@ -16,6 +18,15 @@ abstract class VariantIndependentBaseMainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
         initViews()
+
+        Cockpit.setOnFontSizeChangeListener(
```

Comment:
maybe it would be better to provide `add..ChangeListener` instead of `set..ChangeListener`?